tests/openvino/native/test_pattern_manager.py

```diff
@@ -32,6 +32,9 @@
     HWFusedPatternNames.LINEAR_BATCH_NORM_ACTIVATIONS: "Not relevant for OpenVINO.",
     HWFusedPatternNames.LINEAR_BATCH_NORM_SCALE_SHIFT_ACTIVATIONS: "Not relevant for OpenVINO.",
     HWFusedPatternNames.LINEAR_SCALE_SHIFT_ACTIVATIONS: "Not relevant for OpenVINO.",
+    HWFusedPatternNames.LINEAR_ARITHMETIC_ARITHMETIC: "Not relevant for OpenVINO.",
```

Comment:
Could you explain why LINEAR_ARITHMETIC_ARITHMETIC is not relevant to OpenVINO?

Reply:
Added this pattern to OpenVINO as well